### DISCUSSION TITLE
prompt variables?

### DISCUSSION THREAD

**Discussion by werkwolkbvba** [infiniflow/ragflow#1281] [Q&A]
The only prompt variable i found is {knowledge}. I tried {text} or {input}, but getting errors. they dont mean anything "ERROR: ", based on the fact i only get the error when i use a bad variable, probably excludes other reasons

**Comment by KevinHuSh** [comment]
This variable is used through API, like：
htpp://xxxx/v1/api/completion?gender=male&age=35
The prompt is like:
If {gender} is male, refuse them like 'Sorry, ....'
If {age} is larger than 20, greet her.
